website/docs/r/google_service_account_key.html.markdown

```diff
@@ -0,0 +1,78 @@
+---
+layout: "google"
+page_title: "Google: google_service_accout_key"
+sidebar_current: "docs-google-service-account-key"
+description: |-
+  Allows management of a Google Cloud Platform service account Key Pair
+---
+
+# google\_service\_account\_key
+
+Allows management of a key, and must be created or imported for use with
+[Google Cloud Platform service account](https://cloud.google.com/compute/docs/access/service-accounts).
+
+
+## Example Usage, creating a new Key Pair
+
+```hcl
+resource "google_service_account" "acceptance" {
+  account_id = "%v"
+  display_name = "%v"
+}
+
+resource "google_service_account_key" "acceptance" {
+  service_account_id = "${google_service_account.acceptance.id}"
+  public_key_type = "TYPE_X509_PEM_FILE"
+}
+```
+
+## Create new Key Pair, encrypting the private key with a PGP Key
+
+```hcl
+resource "google_service_account" "acceptance" {
+  account_id = "%v"
+  display_name = "%v"
+}
+
+resource "google_service_account_key" "acceptance" {
+  service_account_id = "${google_service_account.acceptance.id}"
+  pgp_key = "keybase:keybaseusername"
+  public_key_type = "TYPE_X509_PEM_FILE"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+* `name` - The name used for this key pair (not used on create)
+
+* `service_account_id` - (Required) The Serice account id of the Key Pair.
+
+* `key_algorithm` - (Optional) The output format of the private key. GOOGLE_CREDENTIALS_FILE is the default output format. Valid values [ServiceAccountPrivateKeyType](https://cloud.google.com/iam/reference/rest/v1/projects.serviceAccounts.keys#ServiceAccountPrivateKeyType) (only used on create)
```

Comment:
just to make a complete sentence, how about `Valid values are listed at [ServiceAccountPrivateKeyType](...)`